docs/external-node.md

```diff
@@ -0,0 +1,532 @@
+# ExternalNode
+
+## Table of Contents
+
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Name and Namespace](#name-and-namespace)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+  - [Antrea NetworkPolicy configuration](#antrea-networkpolicy-configuration)
+  - [Bypass Antrea NetworkPolicy](#bypass-antrea-networkpolicy)
+- [RBAC for antrea-agent](#rbac-for-antrea-agent)
+- [VM network configuration](#vm-network-configuration)
+- [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [Install Antrea Agent on VM](#install-antrea-agent-on-vm)
+  - [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster)
+  - [Installation on Linux VM](#installation-on-linux-vm)
+    - [Prerequisites](#prerequisites-1)
+    - [Installation](#installation)
+  - [Installation on Windows VM](#installation-on-windows-vm)
+    - [Prerequisites](#prerequisites-2)
+    - [Installation](#installation-1)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+# What is ExternalNode?
+
+`ExternalNode` is a CRD API that enables Antrea to manage the network connectivity
+and security on a Non Kubernetes Node (like a virtual machine or a bare-metal
+server). It supports specifying which network interfaces on the external Node
+are expected to be guarded with Antrea NetworkPolicy rules. The virtual machine
```

Comment:
nit: I'd suggest "protected" instead of "guarded" but feel free to ignore if you prefer the current version

Reply:
update as suggested.